crates/bevy_gltf/src/loader.rs

```diff
@@ -145,7 +151,8 @@ pub struct GltfLoaderSettings {
 impl Default for GltfLoaderSettings {
     fn default() -> Self {
         Self {
-            load_meshes: true,
+            load_meshes: RenderAssetUsages::default(),
+            load_materials: RenderAssetUsages::default(),
```

Comment:
The default for RenderAssetUsages is both. Should we default to only RENDER_WORLD to minimize system RAM usage?